app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<html>
```

Comment:
Do we still need these html tags?

Reply:
I don't think so. Will refactor in US-20.